docs/conf.py

```diff
@@ -107,28 +126,70 @@
 
 # -- Options for HTML output ----------------------------------------------
 
-
-html_theme = 'scanpydoc'
-html_theme_options = dict(
-    navigation_depth=4,
-    logo_only=True,
-    docsearch_index='scanpy',
-    docsearch_key='fa4304eb95d2134997e3729553a674b2',
-)
-html_context = dict(
-    display_github=True,  # Integrate GitHub
-    github_user='theislab',  # Username
-    github_repo='scanpy',  # Repo name
-    github_version='master',  # Version
-    conf_py_path='/docs/',  # Path in the checkout to the docs root
-)
+html_theme = "furo"
+html_theme_options = {
+    "sidebar_hide_name": True,
+    "light_css_variables": {
+        "admonition-font-size": "var(--font-size-normal)",
+        "admonition-title-font-size": "var(--font-size-normal)",
+        "code-font-size": "var(--font-size--small)",
+    },
+}
 html_static_path = ['_static']
+html_css_files = ["css/override.css"]
 html_show_sphinx = False
 html_logo = '_static/img/Scanpy_Logo_BrightFG.svg'
+html_title = "scanpy"
+
+
+# def setup(app):
```

Comment:
We can kill this I guess?

Reply:
I commented that out due to the unused tutorials from the other PR. I should add it back; however, we don't use it in cookiecutter